README.md

```diff
@@ -1 +1,212 @@
-# git
\ No newline at end of file
+<!-- 
+Headings
+Extended this comment over multi lines
+ -->
+
+# Sample .md file
+
+Below are sample snippets that can be used to quickly create/edit `.md` file.
+
+## Table of Contents
+* [Heading](#heading)
+* [Comments](#comments)
+* [Italics](#italics)
+* [Bold](#bold)
+* [Strikethrough](#strikethrough)
+* [BlockQuote](#blockquote)
+* [Links](#links)
+* [Ordered List](#ordered-list)
+* [Unordered List](#unordered-list)
+* [Inline Code Block](#inline-code-block)
+* [Code Blocks](#code-blocks)
+* [Tables](#tables)
+* [Task lists](#task-lists)
+
+
+
+
+### Heading
+#### Syntax: 
+```
+#H1
+##H2
+###H3
+```
+#### Output:
+## H1
+### H2
+#### H3
+---
+### Comments
+#### Syntax: 
+```
+<!---
+Inside Comment block
+-->
+```
+#### Output:
+<!--
+Inside Comment block
+-->
+
+---
+### Italics
+#### Syntax:
+```
+_ Text _
+```
+#### Output:
+_text_
+
+---
+### Bold
+#### Syntax:
+```
+__ Text __
+```
+#### Output:
+__text__
+
+---
+### Strikethrough
+#### Syntax:
+```
+~~Text~~
+```
+#### Output:
+~~Text~~
+
+---
+### BlockQuote
+#### Syntax:
+```
+>Statement 1
+>Statement 2
+>Statement 3
+```
+#### Output:
+>Statement 1 \
+>Statement 2 \
+>Statement 3
+
+---
+### Links
+#### Syntax:
+```
+[Display_Link_Name](link_pathname "hoverover_text")
+```
+#### here we can use reference to the path and mention the path at the end of the file for clean code and better readiblity
+#### Syntax:
+```
+[reference_id]:[link]
+```
+#### Output:
+[GitHub][github-link]
+
+
+---
+### Unordered List
+#### Syntax:
+```
+* Item 1
+* Item 2
+* Item 3
+```
+#### Output:
+* Item 1
+* Item 2
+* Item 3
+
+---
+### Ordered List
+#### Syntax:
+```
+1. Item 1
+2. Item 2
+3. Item 3
+```
+#### Output:
+1. Item 1
+2. Item 2
+3. Item 3
+
+---
+### Inline Code Block
+#### Syntax:
+```
+`Inline code`
+```
+#### Output:
+`Inline code`
+
+---
+### Image
+#### Syntax:
+```
+![image](path/reference)
+```
+
+#### here we can use reference to the path and mention the path at the end of the file for clean code and better readiblity
+#### Syntax:
+```
+[reference_id]:[path]
```

Comment:
Talk about when would you use this syntax: `<img src="images/chandler_in_stormforge.png" width="60%" height="30%">`